django-stubs/views/generic/list.pyi

```diff
@@ -15,14 +15,14 @@ class MultipleObjectMixin(Generic[_M], ContextMixin):
     paginate_by: int
     paginate_orphans: int
     context_object_name: str | None
-    paginator_class: type[Paginator]
+    paginator_class: type[Paginator[_M]]
     page_kwarg: str
     ordering: str | Sequence[str] | None
     def get_queryset(self) -> _SupportsPagination[_M]: ...
     def get_ordering(self) -> str | Sequence[str] | None: ...
     def paginate_queryset(
         self, queryset: _SupportsPagination[_M], page_size: int
-    ) -> tuple[Paginator, Page, _SupportsPagination[_M], bool]: ...
+    ) -> tuple[Paginator[_M], Page[_M], _SupportsPagination[_M], bool]: ...
```

Comment:
In a few cases, I took a leap of faith and decided to `Paginator[_M]` rather than `Paginator[Model]`.

Reply:
Yep these really improve our typing accuracy, thanks!